app/routes.py

```diff
@@ -9,21 +9,38 @@ def get_posts():
 
 
 @app.route('/addPost',methods=['POST'])
-def add_post():
+def addPost():
     post_data=request.get_json()
     title=post_data.get('title')
     body=post_data.get('body')
 
     post=Post(title,body)
     post.save()
-    return "done"
+    return jsonify({"message":"Successfully added"})

```

Comment:
This should also return a http status code 
```suggestion
    return jsonify({"message":"Successfully added"}),  201

```

This link will guide you on the proper status codes to use .
 [List of all status codes](https://www.restapitutorial.com/httpstatuscodes.html)

[Http Methods and expected status codes](https://www.restapitutorial.com/lessons/httpmethods.html)